app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<h2>People list </h1>
+
+
+
+<%@request.each do |request|%>
+
+
+  
+   <h2> <%=request.user.name%></h2>
+    
+    <%= button_to "Accept",friendship_path(request,status:1),method: :put%>
+    
+   <%= button_to "Decline",friendship_path(request,status:0),method: :put%>
+
```

Comment:
**Optional** changing the button's colors depending on the action is a very user-friendly feature that you can add.